charts/prefect-server/values.schema.json

```diff
@@ -200,6 +200,22 @@
             "type": "object"
           }
         },
+	"args": {
+          "type": "array",
+          "title": "Custom container command arguments",
+          "description": "array with arguments",
+          "items": {
+            "type": "string"
+          }
+        },
+	"command": {
+          "type": "array",
+          "title": "Custom container entrypoint",
+          "description": "array with container entrypoint",
+          "items": {
+            "type": "string"
+          }
+        },
```

Comment:
I _think_ the alignment is a bit off here - the `args` and `command` keys themselves need to be indented to the right a bit.

Reply:
Good catch, there was a tab vs space issue... thank you.